doc/reference/guidelines.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: python
+
+=================
+Coding Guidelines
+=================
+
+Python
+======
+
+- PEP8 ignore E501
+- prefer % over format
```

Comment:
What about using a dict: "%(new_name)s is the new %(old_name)s" % {'new_name': 'odoo', 'old_name': 'openerp'}